docs/api-reference/mapbox/overview.md

```diff
@@ -13,8 +13,9 @@ This module makes it easy to use deck.gl as native layers and controls in the Ma
 ### Include the Standalone Bundle
 
 ```html
-<script src="https://unpkg.com/deck.gl@^8.1.0/dist.min.js"></script>
-<script src='https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js'></script>
+<script src="https://unpkg.com/deck.gl@^8.9.0/dist.min.js"></script>
+<script src='https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.js'></script>
+<link href='https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.css' rel='stylesheet' />
```

Comment:
You bring up some good points to consider. We're currently focused on supporting both libraries with the same integration module since they're effectively drop-in replacements for most use cases.

Doing a hard split would add maintenance overhead to consider as well, since there are many things to test and most of the code would remain the same between the modules.

I agree showing both library options is probably more useful at this stage, as well as tracking the differences and limitations common to both, and unique to each.